pkg/services/postgresql/all_in_one_provision.go

```diff
@@ -22,10 +22,7 @@ func (a *allInOneManager) ValidateProvisioningParameters(
 				"*postgresql.AllInOneProvisioningParameters",
 		)
 	}
-	return validateServerParameters(
-		pp.SSLEnforcement,
-		pp.FirewallRules,
-	)
+	return validateDBMSProvisionParameters(&pp.DBMSProvisioningParameters)
```

Comment:
This just felt a little cleaner. If the list of params grows, we won't have to touch this code again.